R/pipeline-helpers.R

```diff
@@ -87,3 +85,84 @@ repeat_add_columns <- function(x, y) {
     full_join(mutate(y, UNIQUE_JOIN_FIELD = 1), by = "UNIQUE_JOIN_FIELD") %>%
     select(-UNIQUE_JOIN_FIELD)
 }
+
+#' Change a particular ISO code in a table to another code.
+#'
+#' A few countries have changed their ISO codes over time.  Usually this is because countries
+#' split or merged; occasionally it's because they just didn't like the old one.  This function
+#' allows us to insert an ISO code change into a pipeline.
+#'
+#' If you're dealing with a split or a merger, then making this change will generally create
+#' duplicate rows, so you should follow it up with an appropriate aggregation.
+#'
+#' @param d The data to be transformed
+#' @param from The old ISO code
+#' @param to The replacement ISO code
+#' @param col The name of the columns with the ISO codes (default = 'iso')
+#' @return Tibble with the old iso code replaced by the new one.
+#' @export
```

Comment:
Since unexported functions can't be used in examples (including examples for the function itself!) or, apparently, tests, I've left them exported for now.  We can come back to this in a cleanup patch.